packages/core/src/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -0,0 +1,173 @@
+import { cloneDeep, isEqual } from "lodash";
```

Comment:
We should import the `lodash` methods one-by-one to reduce the bundle size. Like so:
`import cloneDeep from "lodash/cloneDeep";`
`import isEqual from "lodash/isEqual";`